src/instructlab/model/train.py

```diff
@@ -360,6 +357,13 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="By default, checkpoints are saved at the end of each training epoch. This option disables this behavior.",
 )
+@click.option(
+    "--pipeline",
+    type=click.Choice(["simple", "full", "accelerated"]),
+    default="accelerated",
+    help="which model fidelity pipeline to use when running training. simple uses an SFTTrainer on Linux and MLX on MacOS. This version of training produces low-fidelity models quickly."
```

Comment:
nit: Capitalize the first word to be consistent with the other help text(s). I also suggest changing wording slightly to:
`Model fidelity pipeline for training: 'simple' uses SFTTrainer on Linux or MLX on MacOS, producing low-fidelity models quickly for rapid prototyping. 'full' employs CPU and MPS optimized InstructLab fine-tuning, generating medium-fidelity models over a longer period. 'accelerated' utilizes GPU acceleration and distributed training, yielding high-fidelity models but requiring more time. Choose based on your hardware, time constraints, and desired model quality.`

Reply:
sure

Reply:
done